packages/amazon-ssm-agent/Cargo.toml

```diff
@@ -14,3 +14,6 @@ sha512 = "b614803911b5f248dff6882f58da7a37d0d0397ea531f1f9cf30f52762ff2e80a7f0e7
 
 [build-dependencies]
 glibc = { path = "../glibc" }
+
+[dependencies]
+bash = { path = "../bash" }
```

Comment:
What's this about? I don't like this change. `bash` is only for the `*-dev` variants; it's not allowed as a package dependency.

What I'm not seeing here that I expected to see based on previous discussions is a patch so SSM agent doesn't need bash to execute commands. Ideally that would be some sort of environment or config-driven toggle, and the default would be to not require bash (because it's not present on production variants).

Reply:
When testing, I checked this and saw that `bash` is present in the `aws-dev` variant only and NOT present in the `ecs` variants:

*I enabled admin container and disabled control container before SSHing into the instances.*

```
## aws-dev variant instance
 
[ec2-user@admin]$ sudo sheltie
bash-5.1# find /usr/bin -name "bash" 
/usr/bin/bash
bash-5.1# 
 
## ecs variant instance
 
[ec2-user@admin]$ sudo sheltie
bash-5.1# find /usr/bin -name "bash"
bash-5.1# 
```

Regarding the SSM agent patch to remove bash dependency, that task is still in the works. An initial effort to do that was unsuccessful, so we need more time to complete that. @arnaldo2792 can provide you with more details on this.